cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsReporterSampler.java

```diff
@@ -77,7 +78,8 @@ protected int retrieveMetricsForProcessing(MetricSamplerOptions metricSamplerOpt
       OffsetAndTimestamp offsetAndTimestamp = entry.getValue();
       _metricConsumer.seek(tp, offsetAndTimestamp != null ? offsetAndTimestamp.offset() : endOffsets.get(tp));
     }
-    LOG.debug("Starting consuming from metrics reporter topic partitions {}.", _currentPartitionAssignment);
+    Set<Integer> partitionIds = _currentPartitionAssignment.stream().map(TopicPartition::partition).collect(Collectors.toSet());
```

Comment:
Can we consider using `List<Integer> partitionIds` and sort integers in ascending order? I think that would be more readable.